package/scripts/windows/quarto.cmd

```diff
@@ -57,7 +57,7 @@ IF EXIST "!QUARTO_TS_PATH!" (
 		echo !DENO!>"!DENO_VERSION_FILE!"
 	)
 
-	SET QUARTO_CACHE_OPTIONS=""
+	SET QUARTO_CACHE_OPTIONS=
```

Comment:
🤦

Reply:
If we keep this option, we probably need to also remove quotes at 
https://github.com/quarto-dev/quarto-cli/blob/bd3bd374a7b85cc10f5df2845bb33c44653f33b0/package/scripts/windows/quarto.cmd#L83

Should be 
`	SET QUARTO_CACHE_OPTIONS=--cached-only`

But I was not sure if we want to keep this `QUARTO_CACHE_OPTIONS` or just put back previous behavior with cache. 

Though TBH since we build an exe file for windows, I am really not sure that the second part of the if (if `quarto.ts` does not exists) is used. (unless for conda build ? 🤔 )

Anyhow, not related to this PR